dll/win32/mapi32/version.rc

```diff
@@ -31,6 +31,9 @@
 /* UTF-8 */
 #pragma code_page(65001)
 
+#ifdef LANGUAGE_CY_GB
+    #include "lang/Cy.rc"
+#endif
```

Comment:
```suggestion
```
This needs to be done instead systematically for all modules that have the welsh language translation (it's not limited to just mapi32); besides we don't declare anywhere else the `LANGUAGE_CY_GB` define. If you are interested to do this, do it in a separate specific PR.